recommender/__init__.py

```diff
@@ -1,9 +1,10 @@
-# pylint: disable=too-few-public-methods, unused-argument
+# pylint: disable=too-few-public-methods, unused-argument, unused-variable
 
 """Flask recommender factory"""
 
 import os
 
+import typer
```

Comment:
There is no module named "typer" in our env. I see that this module is not used anywhere.

Reply:
Ah, yes - I wanted to use it instead of `click` (typer is build on top of `click` but provides more elegant strongly typed interface) but flask cli written with `click` in mind, so I had to abandon the idea. It's a remnant of that.